simple_history/management/commands/clean_old_history.py

```diff
@@ -0,0 +1,72 @@
+from django.utils import timezone
+from django.db import transaction
+
+from . import populate_history
+from ... import models, utils
+from ...exceptions import NotHistoricalModelError
+
+
+class Command(populate_history.Command):
```

Comment:
Don't love inheriting from another command here. Would prefer you refactor the shared functions into a shared commands `utils.py`

Reply:
I'm kinda Following the same pattern made to  ``clean_duplicate_history.py``, it also inherit from ``populate_history.Command``